src/test/java/com/linagora/james/mailets/json/MailParserTest.java

```diff
@@ -67,13 +62,14 @@ public void toJsonAsStringShouldParseIntoEmptyJsonWhenEmptyMail() throws Excepti
 
     @Test
     public void toJsonAsStringShouldParseWhenSimpleTextMessage() throws Exception {
-        MimeMessage message = new MimeMessage(Session.getDefaultInstance(new Properties()));
-        message.addFrom(new InternetAddress[] { new InternetAddress("from@james.org", "From"), new InternetAddress("from2@james.org") });
-        message.setRecipients(RecipientType.TO, new InternetAddress[] { new InternetAddress("to@james.org"), new InternetAddress("to2@james.org", "To2") });
-        message.setRecipients(RecipientType.CC, new InternetAddress[] { new InternetAddress("cc@james.org"), new InternetAddress("cc2@james.org", "CC2") });
-        message.setRecipients(RecipientType.BCC, new InternetAddress[] { new InternetAddress("bcc@james.org"), new InternetAddress("bcc2@james.org", "Bcc2"), new InternetAddress("bcc3@james.org") });
-        message.setSubject("my subject");
-        message.setContent("this is my body", "text/plain");
+        MimeMessage message = MimeMessageBuilder.mimeMessageBuilder()
+            .addFrom(new InternetAddress("from@james.org", "From"), new InternetAddress("from2@james.org"))
```

Comment:
the point of builder are to simplify the use of the API.
I won't say that InternetAddress adds anything here and it makes the reading harder.
But that's not important, do as you want.